src/app.html

```diff
@@ -11,8 +11,28 @@
     />
     <title>Good First Issue Finder</title>
     %svelte.head%
+    <style>
+      body {
+        padding: 25px;
```

Comment:
We're using tailwind in this project, so ne have no need for custom stylings inside a style-tag